client/datatypes.go

```diff
@@ -2,9 +2,9 @@
 // User Types
 //
 // Generated with goagen v0.2.dev, command line:
-// $ goagen
+// $ goagen.exe
```

Comment:
Sure, I would put those as part of the regen commit. Unrelated to any 'real' change, just a by product of using latest master of the gen framework. That should go away as soon as goa is 1.0 and we vendor it